src/components/OSCALEditableFieldActions.js

```diff
@@ -18,14 +18,12 @@ export default function OSCALEditableFieldActions(props) {
             : `save-${props.editedFieldPath}`
         }
         onClick={() => {
-          if (props.onFieldSave.length === 6) {
+          if (props.onFieldSave.length > 0) {
```

Comment:
I haven't looked in to when we'd call `onFieldSave` without arguments, but we shouldn't pin to the number of arguments if we can avoid it as that may change.